robot/basestation/static/js/pages/arm.js

```diff
@@ -3,6 +3,10 @@
 // update odroid rx data every second
 // setInterval(updateOdroidRx, 1000);
 
+// Button coloring
+const orange = 'rgb(255, 151, 44)'
```

Comment:
Conventionally constants should be named in all caps, with underscores separating different words

Reply:
Fixed